### PR TITLE
Fix log messages

### PR DESCRIPTION
**Pull request by AnatoliB**
Fixing typos, replacing"FunctionApp" with "function app", etc.

### CODE REVIEW

File: src/resources/PowerShellWorkerStrings.resx [hunk 1]
```diff
@@ -194,16 +194,16 @@
...
   </data>
   <data name="FailToResolveHomeDirectory" xml:space="preserve">
-    <value>Fail to resolve '{0}' path in App Service.</value>
+    <value>Failed to resolve '{0}' path in App Service.</value>
   </data>
   <data name="InstallingFunctionAppDependentModules" xml:space="preserve">
-    <value>Installing FunctionApp dependent modules to '{0}'.</value>
+    <value>Installing function app dependent modules to '{0}'.</value>
   </data>
   <data name="FailToClenupModuleDestinationPath" xml:space="preserve">
```
Comment (Francisco-Gamino) [inline src/resources/PowerShellWorkerStrings.resx]: I don't think this resource string is being used anymore. Please remove it.

Comment (AnatoliB) [reply src/resources/PowerShellWorkerStrings.resx]: I can actually see it being used: https://github.com/Azure/azure-functions-powershell-worker/blob/0be9d316c580c8a9d95345b5d6432fb5478d7835/src/DependencyManagement/DependencySnapshotInstaller.cs#L47

Comment (AnatoliB) [reply src/resources/PowerShellWorkerStrings.resx]: The message needs to be fixed though: these are not _dependent_ modules, it's the other way around.

Comment (Francisco-Gamino) [reply src/resources/PowerShellWorkerStrings.resx]: > The message needs to be fixed though: these are not dependent modules, it's the other way around.

You are right. We should fix the message.

The string I was referring to remove is ```FailToClenupModuleDestinationPath```.

Comment (AnatoliB) [reply src/resources/PowerShellWorkerStrings.resx]: Yes. Removing.

Comment (Francisco-Gamino) [reply src/resources/PowerShellWorkerStrings.resx]: Thank you.

**Pull request merged**